selftests/unit/test_xunit.py

```diff
@@ -56,7 +56,7 @@ def tearDown(self):
         os.remove(self.tmpfile[1])
         shutil.rmtree(self.tmpdir)
 
-    def testAddSuccess(self):
+    def test_add_success(self):
```

Comment:
Ok, only occurrences I see left are in the test script codes present in `unit/test_loader.py`, but I don't think we should touch that.